cmd/edge-proxy/main.go

```diff
@@ -90,7 +96,34 @@ func main() {
 		}
 	}
 
-	server.StartHTTPTunnel(httpTunnelAddr, externalHTTPProxyURI)
+	enableHTTPSTunnel := false
```

Comment:
Can we move lines 99-116 to the section above where the rest of the command line arguments are validated?  It makes sense to group CLI argument checks together, plus it would be strange to spin up a EdgeProxyReverseTunnel on line 93 only to kill it here if we os.exit.

Reply:
Done.